src/classes/EP_TaskDependency_TEST.cls

```diff
@@ -0,0 +1,232 @@
+/*
```

Comment:
* I suggest to annotate test methods with `@IsTest` for better readability.
* Classes and triggers should use the `UpperCamelCase` naming convention. I suggest to rename accordingly.